### DISCUSSION TITLE
[预览]v1.12.9

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#1890] [分享 / Share]
- 修复`首页过滤`选项有时候会丢失的问题. (#1839)
- Safari 兼容性改进:
  - 修复`自定义顶栏`无法点击历史搜索.
  - 修复`简化评论区`的遮罩透明度不正确.
- `自定义顶栏`将在视频关灯时自动隐藏, 与原版顶栏保持一致.
- 修复直播间画中画有时失效的问题. (#1849)
- 移除 `YouTube` 快捷键预设中的 <kbd>Shift + L</kbd> 点赞, 此快捷键已转变为官方快捷键. (#1646)
- 修复宽度大于 1080px 时`强制固定顶栏与侧栏`出现错位. (#1888)
- `传统连播模式`:
  - 修复在合集类页面仅生效一次.
  - 多 P 视频支持在最后 1P 自动关闭连播. (合集类暂时没啥办法检测)
- 新增功能`禁用高赞弹幕样式`和`禁用UP主弹幕样式`. (#1228)

👍**感谢 PR 作者**
- [FoundTheWOUT](https://github.com/FoundTheWOUT): PR #1872

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.9-preview'>[预览]v1.12.9</a>.</em>

**Comment by Geeyun-JY3** [comment]
B 站新加的这个点赞快捷键还不支持长按一键三连~~这个 B 站就是逊啦~~，YouTube 预设的 <kbd>Shift</kbd>+<kbd>l</kbd> 可能还不能直接去掉。

**Reply by the1812** [reply]
那就得换一种键位了, 不然会叠出两次点赞
